db/Pos/DB/Functions.hs

```diff
@@ -7,7 +7,9 @@ module Pos.DB.Functions
        (
        -- * Encoded putting/getting
          dbGetBi
+       , dbGetBiWithVersion
        , dbPutBi
+       , dbPutBiWithVersion
```

Comment:
I think `dbGetBiWithVersion` should be called `dbGetBi` and `dbPutBiWithVersion` should be called `dbPutBi`, because these are the functions we should use and we probably never want to use current `dbGetBi` and `dbPutBi` (w/o tag).
If we really want to use versions without tag, they can have longer names (e. g. `dbGetBiNoVersion`).
I. e. functions with tags are the most useful and should have shorter names, while functions w/o tags are (almost) useless and should have long intimidating names (because probably nobody should use them).